### ISSUE TITLE
Location of preprocessing conversion script

### ISSUE THREAD

**Issue by epreble** [allykgray/DetectionTutorialSpaceNet#1]
In reference to your data preprocessing section, where you mention:
`The BEE-Sharp script convert_Kaggle.py (check file name) converts the coordinate data to pixels.`

 Do you have this script that can be uploaded to this repo, or is there another repo that has this script?

Thanks!

**Comment by sani1486** [comment]
Hello , @epreble 

Did you manage to find the file you were looking for or any other way to convert the labels into coordinates ?

**Comment by epreble** [comment]
Nope. No replies from nVidia. I moved on to tensorflow, which has a much more active community, so it's easier to get things running and get assists on things that don't work.

**Comment by jon-barker** [comment]
@sani1486 @epreble Sorry that a response to this issue fell through the gaps.  The exact pre-processing scripts we used internally are not shareable, but I can point you in the right direction for scripts that do the same thing.

There are a collection of SpaceNet challenge utilities here: https://github.com/SpaceNetChallenge/utilities

These include scripts to convert the raw geoTIFF data into other formats such as CSV.  At that point you should be able to easily convert the data to the Kitti labelling format as described here: https://github.com/NVIDIA/DIGITS/blob/digits-4.0/digits/extensions/data/objectDetection/README.md#label-format

If you try to do this and hit any hurdles let me know and I will try to help.
